index.html

```diff
@@ -79,6 +87,80 @@ <h1>Examples</h1>
   }
 }
       </pre>
+      <section>
```

Comment:
This is an example of the respec plugin.  What is the graph for?  Was the idea this would be autogenerated?  It doesn't match the example data above.